docs/DeveloperGuide.md

```diff
@@ -686,37 +744,203 @@ A successful selection displays the event details.
+
+**Actor:** User
+
+**Preconditions:** User has added one or more contacts to the global contact list or one or more events.
+
+**Guarantees:**
+A successful clear empties all entries in the app.
+
+**MSS:**
+1. User requests to clear all entries, meaning all contacts and events.
+2. Eventy clears all entries in app. </br>
+   *Use case ends*
+
+### Exit the program
+
+**Use case: UC14 - Exits the program**
+
+**Actor:** User
+
+**MSS:**
+1. User requests to exit program.
+2. Eventy terminates. </br>
+   *Use case ends*
+
+### List all contacts or invitees
+
+**Use case: UC15 - List invitees or contacts**
+
+**Actor:** User
+
+**Preconditions:** User has added one or more contacts to the global contact list or invited one or more contacts to the selected event.
+
+**Guarantees:**
+A successful list command lists the contacts of invitees in the app.
+
+**MSS:**
+1. User requests a list of all persons.
+2. Eventy either displays list of global contacts or invitees of an event, depending on whether an event is selected. </br>
+   *Use case ends*
+
+### Edit contact email
+
+**Use case: UC16 - Edit email of contact**
+
+**Actor:** User
+
+**Preconditions:** User has added one or more participant to the global contact list.
+
+**Guarantees:**
+A successful edit updates the person's email in both the global contact list and all the events he participates in.
+
+**MSS:**
+1. User enters index of contact he wants to edit, along with the new email.
+2. Eventy updates the email of the contact. </br>
+   *Use case ends*
+
+**Extensions:**
+
+1a. The participant index provided by the User is invalid.
+- 1a1. Eventy displays an error message to alert the User. </br>
+  *Use case ends*
+
+### Edit contact address
+
+**Use case: UC17 - Edit address of contact**
+
+**Actor:** User
+
+**Preconditions:** User has added one or more participant to the global contact list.
+
+**Guarantees:**
+A successful edit updates the person's address in both the global contact list and all the events he participates in.
+
+**MSS:**
+1. User enters index of contact he wants to edit, along with the new address.
+2. Eventy updates the address of the contact. </br>
+   *Use case ends*
+
+**Extensions:**
+
+1a. The participant index provided by the User is invalid.
+- 1a1. Eventy displays an error message to alert the User. </br>
+  *Use case ends*
+
+### Edit contact tags
+
+**Use case: UC18 - Edit tags of contact**
+
+**Actor:** User
+
+**Preconditions:** User has added one or more participant to the global contact list.
+
+**Guarantees:**
+A successful edit updates the person's tags in both the global contact list and all the events he participates in.
+
+**MSS:**
+1. User enters index of contact he wants to edit, along with the new tags.
+2. Eventy updates the tags of the contact. </br>
+   *Use case ends*
+
+**Extensions:**
+
+1a. The participant index provided by the User is invalid.
+- 1a1. Eventy displays an error message to alert the User. </br>
+  *Use case ends*
+
```

Comment:
Having one for each field seems excessive, could you try thinking of new ones? There can be use cases with more depth.